### ISSUE TITLE
MultiLineLambdaClosingNewline didn't trigger correctly on line which was one character too long

### ISSUE THREAD

**Issue by Smaug123** [fsprojects/fantomas#2642]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/main?data=N4KABGBEDGD2AmBTSAuKBbBBXANosAYrLGALwA6AdhBHgC5gAOAhgE50CMYAFDgJaVEAZzBohdVgIDmYIYgCOASlFgBDCtRpb%2BgoVS1aAPgD4wAZQUA6dM0Y8AZlmoiAtKZas5PIZYCCQsxZqbkVQ-QMIE3Mrez4cOkRWBycHPk86AFEcewAaWUQ4Sngs%2B2U3cIitRxwcAE8AYVhKOmYBEVj0kvzC4uywQ0MwarrG5tbKETkero7xEoqDRQWjUwt5SzxKKToAC0gc8Cg5OjppIVQwAG1DiFAIyAASOlrGZDRINX2bmkeAN2YcFg3ldvloAAwHSpQACSRUQzTMfAAXshIZUACygsAAXW%2BAF80bcsY9nq8Lh9ml97g9-oDgdcoRxCQZIABZZgADwAMgJEFz4ds9sytBwAExgrG4rQE753FlPF7AyDw%2BAAeXsPMEZmeeCp8tpQIuDMqouFPwyRXVmtRWIgkGykEl%2BOZcq0JMV5IARsQcHq3TSAYa0MaIgBmM122FydgEAQAgByiAA7jobVCJECndKXcSFWT3t7YL6zX9A-TbWB0RGoIFmNBEAAhRD2WCsRAABTYzHQiASrD9BgziCzNBlWldPzzSsLxdzBvLUIArNXILX602W22ubAk4loMw5LDfrB96cmgOtEORxAxzQJ3ap16fReoAG6UaKwA2FdrxvN1uIAAqowrysPuh6UMep58Oe1b2ACcjXmAt5EtSpLTs%2BJZvkGIJQgA7D%2BLDrv%2Bbb1DgB5CKM4isFg0B0K2L4QPBOCIREUqjjmaEegWmFzmWH5QgAHIRdZ-puiCsog6CeokjFDAhw5sc6sq5uhT5Fi%2BpbvsGFYAJwicR4mNDgsEVsxrEGOxN6cfKak8RpWHzgJlQcBCFarkRiC%2BPYfaNOgNhyVeSnZipXH5lAM6adhC4uUy7m-huAEWOgfBwCZlByeZimWcp46qdxEW8dSTk6YyprxZ53l9slqVFqZ6Y0dlWhWchNn%2BnZhUOXx2m4S54YVaJvisLATjwAAIog-ApX2gWNUhKFgPer4dRSdBRSVvURBwVbueyHLQvYAAqOzwmYOytnQADqfDwLsckSsFHGhbZBWret-GlS5y67ZyB3HfCWRyGdF3XbdQpfg9OUhXlYVKp8jkfZtBgcN%2BP37ZQsQcqqoHMPRrAZByjBtkIQgwRl1aCZDzW5Xe%2BXhW9CM9SGyMEWjABKBStvAoN3dW6JUzQLULUt7r0-D3U4czIrCeznOsPA8ZYNJiTqtCCToOc1YcPNbWTit6C4KcqZEKwNgnIkh0eozksVhw%2BnuRzcDy6yhtxLyJtmzNK7QDsXZ0YkAD6SY3XdOvPe1r3i8ViNSzQ4orntQ2sMwtSqqwPLiDz4NCQLEBC7rD4rVH%2BoxxWopxVCbKcknKdpxndCK8rrCq%2Brmu22HMMvfTBvxG7gge7jfaW2S1sxREorlZXNep%2BnfDiC7vfG62nuyd7vvJ-7rBByHewd7TsPksX-obbHECiv1ld7QAamWDYCPA0hZ3JlN76hXdw5So-OePO2X5yBBODomTO%2BRRH47zkvzV%2Bi06YfzWl-T649vp-w5GNWAdAADivYCZE2EKTJoT8Ka5xxDTN%2BEd6aRXgUjLQopUaX1dqmBsJloAAGsGwb2Yb2IQqpKBmG7IgYyStyZmQUlAkWj57KzmjkzMurNK6JhTLyJsdBdzwmHogCasRKB8DPJQXwRRJJNzbg1TMj1rLhz1q9ChEsx4GFFDLKe-ApCUAAZQIBTRERONxlgNsh1YCwiQGMHRs0TFQyep3MhGEupSJtlCUU9sHF9koLjPgvw%2BRNCkAYmSrANECG0WTIxlQsqiJgepSRJdpFQlDG5OhvdrRcm7J6eAzAyKwFJlseRqZglNUFiQ6BB8JHvQqZUUMFdKiQGwYkPgPYxg4AANKIEQIwfx8I6CwjYcwVx2dCkiNMa1cxhdLFFXKTE4Zk8xmMI2cw60QghojSKImcQiB4ALyNrybBxM8ETC6cU-pnUynH1LpUi%2B5y2CJTbGNOe0BJApSSQkeAgEtFNAmtAciycglwR2aEsx4SLHkKOQCoZYZf5jImdClZAJtTDSwNRLAjBtS1F1Bili3S869LEUXT%2B1jv4GFDEgsZ8zFl7Ublk9UFzKBXN5AUraYIiH532ctQ5UTjk2K0KGWhYzKWpToKyBAaZtnMvmocbEaIPhCAIKTC4RS8RAA)

#### Code

```fsharp
module Foo =
    let part1 (lines : string seq) : int =
        lines
        |> Seq.map (fun s -> parse (s.AsSpan ()))
        |> Seq.filter (fun (firstElf, secondElf) ->
            fullyContains firstElf secondElf || fullyContains secondElf firstElf
        )
        |> Seq.length
```
            

#### Result

```fsharp
module Foo =
    let part1 (lines: string seq) : int =
        lines
        |> Seq.map (fun s -> parse (s.AsSpan()))
        |> Seq.filter (fun (firstElf, secondElf) -> fullyContains firstElf secondElf || fullyContains secondElf firstElf
        )
        |> Seq.length
```
            
#### Problem description

I expected the code to remain the same; that is to say, the lambda was exactly one character too long, so the body should have been moved onto a new line.

#### Extra information

- [ ] The formatted result breaks my code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas main branch at 1/1/1990

```fsharp
    { config with
                AlignFunctionSignatureToIndentation = true
                MultiLineLambdaClosingNewline = true }
```

<sub>Did you know that you can ignore files when formatting from fantomas-tool or the FAKE targets by using a [.fantomasignore file](https://fsprojects.github.io/fantomas/docs/end-users/IgnoreFiles.html)?</sub>

**Comment by nojaf** [comment]
Hi Patrick, thanks for the report! This is an interesting case for sure. The lambda fits on the same line, let the lambda be wrapped in parentheses does not. Our check is probably at the lambda level while we might benefit from having it one level higher in this case.

I'll look into this one and see what can be done.

**Comment by nojaf** [comment]
Fix available in [v5.2.0-beta-001](https://www.nuget.org/packages/fantomas/5.2.0-beta-001)
